### DISCUSSION TITLE
@tanstack/vue-query support shallowRef?

### DISCUSSION THREAD

**Discussion by GreatAuk** [TanStack/query#5400] [Ideas]
`useQuery` return `data` is a `Ref` value. if data is a very large list, there's a big loss in performance.

**Comment by RomainLanz** [comment]
PR in progress for V5 release: https://github.com/TanStack/query/pull/5493

**Reply by AlejandroAkbal** [reply]
Why was that closed? It's still a valid issue, no?

**Reply by GabrielHangor** [reply]
Any updates on this one?
